src/api/user/index.js

```diff
@@ -36,10 +37,19 @@ export async function login(email, password) {
  * Sign up by email and return status (true or false)
  *
  * @param {string} email - Email
+ * @param {object} utm - UTM parameters object
  * @returns {Promise<{data: {signUp: boolean}, errors: object[]}>} Response data
  */
-export async function signUp(email) {
-  return api.call(MUTATION_SIGN_UP, { email });
+export async function signUp(email, utm) {
+  const variables = { email };
```

Comment:
its [not good](https://v8.dev/blog/fast-properties) to add properties to object after creation. Declare it with "utm: undefined" instead